sass/_theme_badge.sass

```diff
@@ -2,10 +2,10 @@
   position: fixed
   bottom: 0
   left: 0
+  overflow-y: scroll
```

Comment:
This line appears to be causing an issue.

![screen shot 2018-05-01 at 2 35 24 pm](https://user-images.githubusercontent.com/185043/39494888-f3296432-4d4c-11e8-8d75-34e055a2845b.png)

Reply:
See #626